lib/buildpackutil.py

```diff
@@ -307,6 +308,24 @@ def _get_mono_path(directory, mono_version):
     )
 
 
+def _compose_mono_url_path(mono_version):
+    distrib_id = platform.linux_distribution()[0].lower()
+    if distrib_id != "ubuntu":
+        raise Exception(
+            "Only Ubuntu is supported at present, requested distribution: %s"
```

Comment:
can we use consistent formatting in one method? % vs format, format is preferred. The whole file must be updated actually, but this can be another version, only doing formatting.